support/src/main/java/org/jenkinsci/plugins/workflow/support/actions/NotExecutedNodeAction.java

```diff
@@ -0,0 +1,20 @@
+package org.jenkinsci.plugins.workflow.support.actions;
```

Comment:
:bug: I already recommended this be be in `api/src/main/java/org/jenkinsci/plugins/workflow/actions/`. Why here?